doc/source/myrocks/recover_from_crash.rst

```diff
@@ -0,0 +1,56 @@
+.. _myrocks_recovery:
+
+======================================
+Recovering from a MyRocks Server Crash
+======================================
+
+MyRocks provides multiple crash recovery configurations. Crash recovery is
+accomplished by replaying the transactions from the WAL (Write Ahead Log) files.
+
+You can set the following crash recovery modes in my.cnf to define how WAL files are applied:
```

Comment:
These are for the rocksdb variable wal_recovery_mode.  This variable is set from the MyRocks variable rocksdb_wal_recovery_mode, so, we do not need to discuss the wal_recovery_mode for rocksdb as it is redundant with the description of rocksdb_wal_recovery_mode MyRocks variable.  The original bug report was that our documentation of rocksdb_wal_recovery_mode just states that the value can be from 0 to 3 with no description of what 0, 1, 2, or 3 means.

Reply:
Fixed. I moved some information to the variables page. Removed the Recover page.